### DISCUSSION TITLE
Single report for test run with multiple

### DISCUSSION THREAD

**Discussion by HartleSeb** [thunderclient/thunder-client-support#1220] [Q&A]
Is it possible to generate a single report for a test run with multiple collections instead of generating a report for each collection?

And if not has someone already a workaround or script to do so?

**Comment by rangav** [comment]
Hi @HartleSeb No its not possible, Currently all the reports are based on per collection.

**Reply by HartleSeb** [reply]
This is a pity. Is that somehow planned for a future version or no priority topic?

**Reply by rangav** [reply]
This is not priority now.

But you can create a feature request and lets see how many other users want this feature.

how big is your team that want this feature?

**Reply by HartleSeb** [reply]
I will create a feature 👍  We are a team with round about 20 people at the moment.
